composer.json

```diff
@@ -41,7 +41,7 @@
         "composer/installers": "~1.0",
         "doctrine/annotations": "^1.2",
         "doctrine/collections": "^1.3",
-        "drupal/console-core": "1.8.0",
```

Comment:
Can you revert this changes ? The version should be fixed

Reply:
done